### ISSUE TITLE
USWDS - Bug: Need error alert or suggestions for input mask

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#6119]
### Describe the bug

Currently, if a person is typing  into the [input mask component](https://designsystem.digital.gov/components/input-mask/) and types, for example, a letter instead of a numeral in the SSN example, there is no visible nor audible cue letting the person know of the error.

Currently this issue causes the input mask to fail [WCAG 3.3.3](https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html)

### Steps to reproduce the bug

1. Go to https://designsystem.digital.gov/components/input-mask/
2. Try typing in a letter in the social security number field
3. Listen or look for a cue telling you you're typing in an invalid character

### Expected Behavior

Ideally, there would be instructions above the input mask stating "only numerals allowed" or "only letters allowed" to prevent an error. The example text is helpful but not clear that only a certain type of character is allowed.

Similarly, if a screen reader user mistypes in a character, they are not told audibly of the mistake.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amyleadem** [comment]
Closing this issue as a duplicate of #5481
